src/test/java/org/jboss/hal/testsuite/test/runtime/TransactionsMetricsTestCase.java

```diff
@@ -0,0 +1,73 @@
+package org.jboss.hal.testsuite.test.runtime;
+
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.testsuite.fragment.MetricsAreaFragment;
+import org.jboss.hal.testsuite.fragment.MetricsFragment;
+import org.jboss.hal.testsuite.page.runtime.TransactionsMetricsPage;
+import org.jboss.hal.testsuite.test.category.Shared;
+import org.jboss.hal.testsuite.util.Console;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author mkrajcov <mkrajcov@redhat.com>
+ */
+@RunWith(Arquillian.class)
+@Category(Shared.class)
+public class TransactionsMetricsTestCase {
+
+    public static final String NUMBER_OF_TRANSACTIONS = "Number of Transactions";
+    public static final String COMMITTED = "Commited";
+    public static final String ABORTED = "Aborted";
+    public static final String TIMED_OUT = "Timed Out";
+    public static final String APPLICATION_FAILURES = "Application Failures";
+    public static final String RESOURCE_FAILURES = "Resource Failures";
+    public static final int DELTA = 3;
+    @Drone
+    private WebDriver browser;
+
+    @Page
+    private TransactionsMetricsPage tmPage;
+
+    @Before
+    public void before(){
+        browser.navigate().refresh();
+        Graphene.goTo(TransactionsMetricsPage.class);
+        browser.manage().window().maximize();
```

Comment:
Did you consider to use org.jboss.hal.testsuite.util.Console.maximizeWindow() instead?